addOns/spider/CHANGELOG.md

```diff
@@ -16,3 +16,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+|                                                                                                                                         | Svg - 'href' and 'xlink:href' attributes of 'image' and 'script' elements                                                                                                                                           |
+|                                                                                                                                         | Table - 'background' attribute                                                                                                                                                                                      |
+|                                                                                                                                         | Video - 'poster' attribute                                                                                                                                                                                          |
+| Form - proper handling of Forms with both GET and POST method. The fields values are generated validly, including HTML 5.0 input types. | Form - proper handling of Forms with both GET and POST method. The fields values are generated validly, including HTML 5.0 input types 'form', 'formaction', 'formmethod' attributes of buttons are also respected. |
+| Comments - Valid tags found in comments are also analyzed, if specified in the Options Spider screen                                    | Comments - Valid tags found in comments are also analyzed, if specified in the Options Spider screen                                                                                                                |
+|                                                                                                                                         | Import - 'implementation' attribute                                                                                                                                                                                 |
+|                                                                                                                                         | Inline string - 'p', 'title', 'li', 'h1', 'h2', 'h3', 'h4', 'h5', 'h6', and 'blockquote' tags                                                                                                                       |
+|                                                                                                                                         | SVG image files are parsed to identify HREF attributes and extract/resolve any contained links.                                                                                                                     |
```

Comment:
This one is mentioned in Added, remove that one? IMO all three sections could be removed, automation changes were already released in the Automation add-on and promotion is not relevant, it's a new add-on.

Reply:
Okay, I’ll adjust it.